packages/api/src/command/medical/patient/create-patient.ts

```diff
@@ -69,14 +69,29 @@ export async function createPatient({
       contact,
     },
   };
+
+  let newPatient = await PatientModel.create(patientCreate);
```

Comment:
Let's avoid using variables (and updating objects if possible), please.

We can create a small function to hydrate the patient's address w/ coordinates and that either returns the original param if `addressWithCoordinates` is false, or a new one (the result of `.update)

Reply:
Done.